components/patient-data/ui/src/main/resources/PhenoTips/PatientClass.xml

```diff
@@ -555,6 +555,22 @@
       <values>M=Male|F=Female|O=Other|U=Unknown</values>
       <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
     </gender>
+    <gene_comments>
```

Comment:
`genetic_notes` is better. In general, "notes" is a better term than "comments", and we should rename more of the existing fields to use the `XYZ notes` name pattern.